src/aws.ts

```diff
@@ -31,7 +31,10 @@ const aws = {
   },
   get s3() {
     try {
-      return cache.s3 = cache.s3 || new (require('aws-sdk/clients/s3') as typeof S3)(this.creds)
+      return cache.s3 = cache.s3 || new (require('aws-sdk/clients/s3') as typeof S3)({
+        ...this.creds,
+        endpoint: process.env.AWS_S3_ENDPOINT,
```

Comment:
couldn't this just go on line 29?

Reply:
Ideally yeah, but `this.creds` is also passed to the CloudFront constructor on line 43 and CloudFront also accepts an endpoint option that we wouldn't want to set.

Reply:
ah ok, that makes sense